src/unity/python/turicreate/cython/cy_variant.pyx

```diff
@@ -694,6 +694,20 @@ cdef variant_type from_value(object v) except *:
 
 ################################################################################
 
+cdef read_archive_version(variant_map_type& d):
```

Comment:
Move this function into cy_unity.pyx It doesn't quite belong in cy_variant.

Reply:
Will do!